pkg/question/selectors/environments.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/environments"
 )
 
-func EnvironmentsMultiSelect(ask question.Asker, client *client.Client, s factory.Spinner, message string) ([]string, error) {
```

Comment:
@domenicsim1 
I changed this to return objects rather than ID's because we want a list of environment _names_ in DeployRelease rather than ID's, but I didn't want to break your existing environment code that was using ID's